README.md

```diff
@@ -2,7 +2,7 @@
 
 To get up and running with scaffold, ensure the dependencies are installed:
 
-    npm run-script install-dependencies
+    npm install
```

Comment:
The main issue is that you need to run `npm install` if you were to clone the project (so if you're not the person generating the project). We could put `npm install` in `build-js.sh`, but not sure if it's worth slowing things down for...